src/dirwatcher/Dirwatcher.js

```diff
@@ -0,0 +1,65 @@
+import { EventEmitter } from 'events';
+import { readdirSync, statSync } from 'fs';
+import * as path from 'path';
+
+export class Dirwatcher {
+  static get DIR_CHANGE_EVENT() {
```

Comment:
Why did you decide on placing this value as a static field, even getter method instead of just creating a separate variable like `watcherEvents`?